deployment/terraform/container_service.tf

```diff
@@ -0,0 +1,205 @@
+locals {
+  app_image = "${module.ecr_repository.repository_url}:${var.image_tag}"
+}
+
+#
+# Security Group Resources
+#
+resource "aws_security_group" "alb" {
+  vpc_id = "${module.vpc.id}"
+
+  tags {
+    Name        = "sgLoadBalancer"
```

Comment:
I think that the name of this resource should include some reference to the service behind it. In this case, it could be something like `App` (e.g., `sgAppLoadBalancer`).